### ISSUE TITLE
Make EDA tab error messages more friendly

### ISSUE THREAD

**Issue by chowington** [VEuPathDB/web-eda#1271]
Big error messages like this one (created by io-ts) are somewhat common when an error occurs:

![image](https://user-images.githubusercontent.com/12377649/180026248-10d95a7d-bba0-4db3-8788-57010e39d7d4.png)

We should make these more user-friendly (like the oops error messages), possibly giving them a friendlier message with the error itself semi-hidden in a collapsible.

**Comment by asizemore** [comment]
cool idea from @jernestmyers : add a Read More option to the current banner component.
Doesn't make it perfect but doesn't break the ux

**Comment by asizemore** [comment]
might be already part of the component. Think maybe @moontrip did this work?

**Comment by moontrip** [comment]
@asizemore yes Read more/less feature is currently available at CoreUI Banner as I added it to address other tickets (e.g., Banner message on Scatterplot/Lineplot Vizs, etc.)

**Comment by aurreco-uga** [comment]
based on meeting Jan 18 2023 I understand this is a service formatting error. .not a user error.. therefore we should not show an error banner (for user errors).. but a generic popup?
I dont think this is a styling issue.
